mz_zip.h

```diff
@@ -30,12 +30,12 @@ typedef struct mz_zip_file_s {
     time_t modified_date;        /* last modified date in unix time */
     time_t accessed_date;        /* last accessed date in unix time */
     time_t creation_date;        /* creation date in unix time */
-    uint32_t crc;                /* crc-32 */
```

Comment:
I don't think this structure can be changed due to compatiblity.

Reply:
> I don't think this structure can be changed due to compatiblity.

@nmoinvaz, im fixed this in commit: https://github.com/zlib-ng/minizip-ng/pull/844/commits/83e0e5462ea2dc5dc0579b4632fa92300777bdfe